### ISSUE TITLE
--hide flag not working

### ISSUE THREAD

**Issue by RichardBray** [open-cli-tools/concurrently#294]
Am I doing something wrong? I'm using concurrently v6.3.0 and trying to hide like this:
```json5

  "scripts": {
    "start": "concurrently --hide \"comp-server\" \"npm run watch\" \"npm run server\" \"npm run comp-server\"",
    // ...
  }
    ```

I've also tried it with `concurrently --hide \"2\"` and it still shows the output.

Any ideas?

**Comment by gustavohenke** [comment]
It's not published yet, sorry for the confusion!
See also #293

**Comment by RichardBray** [comment]
Ah sorry my bad, I'll wait for an update before I re-test this feature.

**Comment by gustavohenke** [comment]
It's published as v6.4.0!
